docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -335,6 +335,18 @@ This configuration only impacts resources served from the fixed or static URL, `
 
 For more information, see link:https://quarkus.io/blog/path-resolution-in-quarkus/[Path Resolution in Quarkus].
 
+[[add-security-identity-roles]]
+=== Add `SecurityIdentity` roles
+
+Winning role-based policy can add new roles to the `SecurityIdentity` based on the roles `SecurityIdentity` already possess.
```

Comment:
I believe we should say here that the security identity roles can be mapped to the deployment specific roles, `add` is a side-effect of the mapping process